src/main/java/org/broadinstitute/hellbender/engine/filters/ExcludeIDsVariantFilter.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.util.Set;
+import java.util.HashSet;
+
+/**
+ * Exclude variants with any of these IDs.
+ * Matching is done by case-sensitive exact match.
+ */
+public final class ExcludeIDsVariantFilter implements VariantFilter {
```

Comment:
Name filters using predicate-style names that describe the variants that will be **kept** (rather than filtered out). See `ReadFilterLibrary` for naming examples.